test/e2e/expectedfailures/exit-handler-fail-missing-output.yaml

```
@@ -0,0 +1,39 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: exit-handler-fail-missing-output-
+  labels:
+    test: test
+spec:
+  entrypoint: failure-workflow
+  templates:
+    - name: failure-workflow
+      steps:
+        - - name: step1
+            template: intentional-fail
+            hooks:
+              exit:
+                template: lifecycle-hook
+                arguments:
+                  parameters:
+                    - name: hello-param
+                      value: '{{steps.step1.outputs.parameters.hello-param}}'
+    - name: intentional-fail
+      outputs:
+        parameters:
+          - name: hello-param
+            valueFrom:
+              path: /tmp/hello_world.txt
+      container:
+        image: alpine:latest
+        command: ["sh", "-c"]
+        args: ["echo intentional failure; exit 1"]
+    - name: lifecycle-hook
+      inputs:
+        parameters:
+          - name: hello-param
+      container:
+        image: argoproj/argosay:v1
+        command: [cowsay]
```

```suggestion
        image: busybox
        command: [echo]
```

`argosay` is slated to be removed per https://github.com/argoproj/argo-workflows/pull/11613#issuecomment-1957786530 and #13429, "Future Work" part 1, so let's not add any more usage of it. can use `busybox` with plain `echo` as I did in #13429

---

Thanks for the hint. Updated 🚀